src/main/java/com/google/cloud/genomics/dataflow/functions/AlleleSimilarityCalculator.java

```diff
@@ -58,38 +58,39 @@ public void processElement(ProcessContext context) {
     CallSimilarityCalculator callSimilarityCalculator =
         callSimilarityCalculatorFactory.get(isReferenceMajor(variant));
     FluentIterable<KV<Call, Call>> pairs =
-        PairGenerator.<Call, ImmutableList<Call>>allPairs(getSamplesWithVariant(variant));
+        PairGenerator.<Call, ImmutableList<Call>>allPairs(getSamplesWithVariant(variant), false,
```

Comment:
might consider inlining pairs at its single reference site to avoid having to explicitly mention the FluentIterable type.